README.md

```diff
@@ -1,2 +1,44 @@
-# abrt-dev-modules
-Developer modules for tracking exceptions in software
+<p align="center">
+<a href="https://retrace.fedoraproject.org/">
+    <img alt="ABRT" src="https://github.com/abrt/abrt/blob/master/icons/hicolor_apps_scalable_abrt.svg" width="123">
```

Comment:
```suggestion
    <img alt="ABRT" src="https://github.com/abrt/abrt/blob/master/icons/hicolor_apps_scalable_abrt.svg" width="128">
```

Reply:
It could be slightly blurry if it’s not a power-of-two size.

Reply:
(I would commit and squash, but I don’t think it will be possible to selectively squash)